sdk/rust/src/sqlite.rs

```diff
@@ -1,14 +1,19 @@
+#![allow(missing_docs)]
```

Comment:
I would expect us to want to enforce docs for the public SDK interface?

Reply:
I agree though interestingly if you look at most of the other modules of the sdk they already allow missing docs. It seems we're not getting docs from the code generated by wit-bindgen. I need to look more into what specifically is happening though.

Reply:
Ah, okay, no worries then, we can assess that as s separate issue then.  :shipit: